Allow dumpstate to call automotive_display_service

ceiba1985 opened the pull request:
This is used to fix a CTS failure:
'android.security.cts.SELinuxHostTest#testNoBugreportDenials' the relevant logs:
'java.lang.AssertionError: Found illegal SELinux denial(s):
    avc:  denied  { call } for  scontext=u:r:dumpstate:s0
    tcontext=u:r:automotive_display_service:s0 tclass=binder
    permissive=0'

Tracked-On: OAM-122069

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

ceiba1985 committed:
Allow dumpstate to call automotive_display_service

This is used to fix a CTS failure:
'android.security.cts.SELinuxHostTest#testNoBugreportDenials'
the relevant logs:
'java.lang.AssertionError: Found illegal SELinux denial(s):
    avc:  denied  { call } for  scontext=u:r:dumpstate:s0
    tcontext=u:r:automotive_display_service:s0 tclass=binder
    permissive=0'

Tracked-On: OAM-122069
Signed-off-by: jizhenlo <zhenlong.z.ji@intel.com>

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /cactus-absp-or-local/celadon-merge/639
